packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.1-nullsafety
```

Comment:
Since it's pre-release, you can do `2.0.0-nullsafety.1`

Reply:
done.